lib/documents.js

```diff
@@ -340,6 +341,13 @@ function readDocumentsImpl(contentOnly, args) {
     categories = [categories];
   }
 
+  if (categories != null) {
+    var i = 0;
+    for (i = 0; i < categories.length; i++) {
+      categories[i] = categories[i] === 'metadataValues' ? 'metadata-values' : categories[i];
+    }
+  }
+
```

Comment:
Should the function copy instead of modifying the caller's array?  Possibly a mapper function would be a straightforward way to make the change at the same time as copying?